content/doc/book/managing/users.adoc

```diff
@@ -1,15 +1,129 @@
 ---
 layout: section
-wip: true
+title: Users
 ---
 ifdef::backend-html5[]
-:notitle:
-:description:
+:description: Learn how to manage users in Jenkins, including creating, configuring, and understanding user management.
 :author:
 :sectanchors:
 :toc:
+:toclevels: 4
 :hide-uri-scheme:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
 endif::[]
 
 = Users
 
+Managing users in Jenkins involves creating, configuring, and maintaining user accounts for individuals who interact with the Jenkins instance. This page provides a step-by-step guide on how to manage users effectively.
+
+== What Managing Users Means
+
+In Jenkins, managing users refers to the process of creating, configuring, and maintaining user accounts. This includes:
+
+- Creating new users.
+- Configuring user details (e.g., full name, email address, password).
```

Comment:
```suggestion
- Configuring user details such as full name, email address, and password.
```
Suggestion to replace `e.g.` with `such as`. [e.g should be avoided when creating documentation](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg)